client/app/services/query-result.js

```diff
@@ -5,6 +5,7 @@ import { QueryResultError } from "@/services/query";
 import { Auth } from "@/services/auth";
 import { isString, uniqBy, each, isNumber, includes, extend, forOwn, get } from "lodash";
 
+const JSONbigString = require("json-bigint")({ storeAsString: true });
```

Comment:
This code can be changed `import` style? In order to unify the code style of the entire project, it is recommended to use import to import the package.

Reply:
And change `JSONbigString` to `jsonParse`?